critique.py

```diff
@@ -62,40 +63,49 @@ def isNeXusGroup(obj, NXtype):
 
 class Critic(object):
     '''
-    describe a file in terms of NeXus compliance
+    describe a file in terms of NeXus compliance.
+    Each method starting with "test_" will contribute a column to the results table.
     
+    :param str path: absolute or relative path to the file directory
     :param str fname: (absolute or relative path and) name of file
     '''
     
     def __init__(self, path, fname):
```

Comment:
consider changing to `path=None, fname=None` with consequence to the caller as shown below